.github/workflows/acc-test.yml

```diff
@@ -0,0 +1,44 @@
+name: acct-test
+on: 
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  pr-acc-test:
+    if: ${{github.repository == 'huaweicloud/terraform-provider-huaweicloud' &&
```

Comment:
I think we should not harding coding the repository address, suggest to remove it

Reply:
done